### DISCUSSION TITLE
Is playground functional

### DISCUSSION THREAD

**Discussion by ThomasTSWD** [dargmuesli/nuxt-cookie-control#223] [General]
Hi there ! 

I wonder if the playground is functional ? I don't see javascript to accept or deny cookies.

Maybe target cookie id does the job (and dont need a javascript logic ?)

Hope u could answer me !

Best

**Comment by dargmuesli** [comment]
On the current master it looks as it should, I'd say.

![image](https://github.com/user-attachments/assets/91a077ee-9582-4f1e-bd81-9a18771f0392)

**Comment by ThomasTSWD** [comment]
Thank's for your answer.

I realized the javascript logic to block cookies is in module.ts if iam right.

Best regards
